modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/index/ResumeCreateIndexTest.java

```diff
@@ -56,10 +60,19 @@
     @Override protected IgniteConfiguration getConfiguration(String igniteInstanceName) throws Exception {
         return super.getConfiguration(igniteInstanceName)
             .setCacheConfiguration(
-                cacheCfg(DEFAULT_CACHE_NAME, null).setAffinity(new RendezvousAffinityFunction(false, 1))
+                cacheConfig(DEFAULT_CACHE_NAME),
+                cacheConfig(DEFAULT_CACHE_NAME + 2)
```

Comment:
Maybe I don’t understand something, but why cache2 if it is just filling up?

Reply:
We need non-empty cacheGroup after the first one will be destroyed.

Reply:
Okay, write a comment (a little more detail) why you need it so that someone else does not delete it.